Config/LocustEquivalentCavity.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
What is the difference b/w this file and LocustCavityEqCircuitTest.json ? 
If they are the same, could remove one of them to avoid confusion in the future.

Reply:
You are correct that these are identical. I think I just used `cp` rather than `mv` when I renamed this file. I've since removed one of the versions to avoid confusion.